component-versions.md

```diff
@@ -2,6 +2,35 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+[Details=1.11.5]
```

Comment:
This one should probably go between 1.11.4 and 1.12.0 - that makes it easier to find, and for the component version, the time when the release came out is less important than for the release notes.